src/helpers/dom/classList.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * @class ClassList
+ */
+class ClassList {
+  constructor(element) {
+    this._element = element;
+  }
+
+  contains(value) {
+    const className = this._element.className;
+    const classLen = className.length;
+    const valueLen = value.length;
+    const isLenDiffer = classLen >= valueLen;
+    let index = classLen > 0 && valueLen > 0 && isLenDiffer ? className.indexOf(value) : -1;
+    let result = index > -1;
+    let nextChar, prevChar;
+
+    while (result && isLenDiffer) {
+      prevChar = className[index - 1];
+      nextChar = className[index + valueLen];
+      result = (!prevChar || prevChar === ' ') && (!nextChar || nextChar === ' ');
+      if (result || !nextChar) {
+        break;
+      }
+      index = className.indexOf(value, index + valueLen);
+      result = index > -1;
+    }
+    return result;
+  }
+
+  add(value) {
+    if (!this.contains(value)) {
+      this._element.className += (this._element.className ? ' ' : '') + value;
```

Comment:
In the official `classList` specification there no such method.
[Element.classList](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList)